### PR TITLE
add instructions for setting up id validation with noid-rails

### PR DESCRIPTION
**Pull request by elrayle**
refs #3128

Provides instructions on how to avoid LDP Conflict/Gone errors by setting up id validation for noid-rails.

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Comment by elrayle** [comment]
@cjcolvar I added the additional information you requested.

File: README.md [hunk 1]
```diff
@@ -279,6 +280,20 @@ And then run `bundle update redis`.
...
+
+Hyrax uses [noid-rails](https://github.com/samvera/noid-rails) for id generation.  By default, the minter will not validate whether a newly minted id is already in use.
+
+To set up id validation, add `config/initializers/noid_rails.rb`
+
+```
+::Noid::Rails.config.identifier_in_use = lambda { |id| ActiveFedora::Base.exists?(id) || ActiveFedora::Base.gone?(id) }
+```
+
+Requirement: noid-rails >= 3.0.1 (older versions prevented the override of identifier_in_use)
```
Comment (jcoyne) [inline README.md]: Shouldn't this be in the gemspec?

Comment (cjcolvar) [reply README.md]: :+1: I don't see any reason to not bump it in the gemspec since it is already at `~> 3.0.0`.

**Comment by no-reply** [comment]
Can we redirect this work to the Hyrax Management Guide, perhaps with re-framing?

There are proposed fixes for the underlying issue in discussion at https://github.com/samvera/hyrax/issues/3128.

**Pull request closed without merging**